src/main/java/wolox/training/models/Users.java

```diff
@@ -0,0 +1,99 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+public class Users {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Long userId;
+
+    @Column(nullable = false)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    @Column(nullable = false)
+    @ManyToMany
+    @JoinTable(name = "book_user",
+            joinColumns = @JoinColumn(name = "userId", referencedColumnName = "userId"),
+            inverseJoinColumns = @JoinColumn(name = "bookId",
+                    referencedColumnName = "bookId"))
+    private List<Book> books = new ArrayList<>();
```

Comment:
Suggestion:
I recommend using Collections.emptyList() instead of new ArrayList<>().
😄